RELEASE.md

```diff
@@ -1,39 +1,39 @@
 # Release Process
-zfs-localpv follows a monthly release cadence. The scope of the release is determined by contributor availability. The scope is published in the [Release Tracker Projects](https://github.com/orgs/openebs/projects/10).
+zfs-localpv follows a on quaterly release cadence for minor version releases. The scope of the release is determined by contributor availability. The scope is published in the [Release Tracker Projects](https://github.com/orgs/openebs/projects/78).
 
-## Release Candidate Verification Checklist
+## Pre-release Candidate Verification Checklist
 
-Every release has release candidate builds that are created starting from the third week into the release. These release candidate builds help to freeze the scope and maintain the quality of the release. The release candidate builds will go through:
+Every release has a prerelease tag that gets created on branch creation, explained further below. This prerelease tag is meant for all the below action items throughout the release process:
 - Platform Verification
 - Regression and Feature Verification Automated tests.
-- Exploratory testing by QA engineers
-- Strict security scanners on the container images
-- Upgrade from previous releases
+- Exploratory testing by QA engineers.
+- Strict security scanners on the container images.
+- Upgrade from previous releases.
 - Beta testing by users on issues that they are interested in.
-- Dogfooding on OpenEBS workload and e2e infrastructure clusters.
 
-If any issues are found during the above stages, they are fixed and a new release candidate builds are generated.
+If any issues are found during the above stages, they are fixed and the prerelease tag is overidden by the newer changes and are up for above action items again.
 
-Once all the above tests are completed, a main release tagged image is published.
+Once all the above tests are completed, a main release is created.
 
 ## Release Tagging
 
-zfs-localpv is released as container image with versioned tag.
+zfs-localpv is released with container images and a respective helm chart as the only recommended way of installation. Even though the [zfs-operator](./deploy/zfs-operator.yaml) is also published, it is generated by templating the helm chart itself.
```

Comment:
Should we replace 'zfs-localpv' with LocalPV ZFS everywhere?

Reply:
Done.